docs/take-to-prod.md

```diff
@@ -0,0 +1,108 @@
+#### Scenario 2. SparkSession code and business logic in the same project, but different .cs files
+This would be the usecase when you have SparkSession code and business logic (UDFs) in the different .cs files but in the same project (e.g. SparkSession in Program.cs, business logic in BusinessLogic.cs and both are in mySparkApp.csproj).
+
+### Package your application - single dependency
+Please follow [Get Started](https://github.com/dotnet/spark/#get-started) to build your application in Scenario 1 and Scenario 2.
+
+### Launch your application - single dependency
+#### 1. Using spark-submit
+Please see below as an example of running your app with `spark-submit` in Scenario 1 and Scenario 2.
+```shell
+%SPARK_HOME%\bin\spark-submit \
+--class org.apache.spark.deploy.dotnet.DotnetRunner \
+--master local \
+--files bin\Debug\netcoreapp3.0\mySparkApp.dll \
+bin\Debug\netcoreapp3.0\microsoft-spark-2.4.x-0.6.0.jar \
+dotnet bin\Debug\netcoreapp3.0\mySparkApp.dll <app arg 1> <app arg 2> ... <app arg n>
+```
+#### 2. Using Apache Livy
+Please see below as an example of running your app with Apache Livy in Scenario 1 and Scenario 2.
+```shell
+{
+    "file": "adl://<cluster name>.azuredatalakestore.net/<some dir>/microsoft-spark-2.4.x-0.6.0.jar",
+    "className": "org.apache.spark.deploy.dotnet.DotnetRunner",
+    "files": [“adl://<cluster name>.azuredatalakestore.net/<some dir>/mySparkApp.dll" ],
+    "args": ["dotnet","adl://<cluster name>.azuredatalakestore.net/<some dir>/mySparkApp.dll","<app arg 1>","<app arg 2>,"...","<app arg n>"]
+}
+```
+
+## How to take your application to production when you have multiple dependencies
+### Scenarios - multiple dependencies
+#### Scenario 3. SparkSession code in one project that references another project including the business logic
+This would be the usecase when you have SparkSession code in one project (e.g. mySparkApp.csproj) and business logic (UDFs) in another project (e.g. businessLogic.csproj).
+#### Scenario 4. SparkSession code references a function from a Nuget package that has been installed in the csproj
+This would be the usecase when SparkSession code references a function from a Nuget package in the same project (e.g. mySparkApp.csproj).
+#### Scenario 5. SparkSession code references a function from a DLL on the user machine
+This would be the usecase when SparkSession code reference business logic (UDFs) on the user machine (e.g. SparkSession code in the mySparkApp.csproj and businessLogic.dll on a different machine). 
+#### Scenario 6. SparkSession code references functions and business logic from multiple projects/solutions that themselves depend on multiple Nuget packages
+This would be a more complex usecase when you have SparkSession code reference business logic (UDFs) and functions from nuget packages in multiple projects and/or solutions.
```

Comment:
```suggestion
This would be a more complex use case when you have `SparkSession` code reference business logic (UDFs) and functions from Nuget packages in multiple projects and/or solutions.
```